src/color-slider/color-slider.js

```diff
@@ -64,8 +64,15 @@ const Self = class ColorSlider extends NudeElement {
 			prop.applyChange(this._el.slider, change);
 
 			let value = change.value;
-			if (this.tooltip === "progress" && (name === "value" || name === "defaultValue")) {
-				value = +(this.progress * 100).toPrecision(4);
+			if (this.tooltip === "progress") {
+				if (name === "value" || name === "defaultValue") {
+					value = +(this.progress * 100).toPrecision(4);
+				}
+				else {
+					// Spinner values when tooltip is "progress"
+					let values = { min: 1, max: 100, step: 1 };
+					value = values[name];
```

Comment:
```suggestion
					value = ({ min: 1, max: 100, step: 1 })[name];
```

(just a suggestion, only accept if you find it more readable)